app/src/main/java/de/westnordost/streetcomplete/quests/police_type/AddPoliceType.kt

```diff
@@ -0,0 +1,32 @@
+package de.westnordost.streetcomplete.quests.police_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+
+class AddPoliceType : OsmFilterQuestType<PoliceType>() {
+
+    override val elementFilter = """
+        nodes, ways, relations with
+          amenity = police
+          and !operator
+    """
+    override val commitMessage = "Add police type"
+    override val wikiLink = "Tag:amenity=police"
+    override val icon = R.drawable.ic_quest_police
+    override val enabledInCountries = NoCountriesExcept(
+        "IT"
+    )
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_policeType_title
+
+    override fun createForm() = AddPoliceTypeForm()
+
+    override fun applyAnswerTo(answer: PoliceType, changes: StringMapChangesBuilder) {
+        changes.add("name", answer.policeName);
+        changes.add("operator", answer.operator.operatorName);
+        if (!answer.operator.wikidata.isNullOrEmpty()) changes.add("operator:wikidata", answer.operator.wikidata);
+        if (!answer.operator.wikipedia.isNullOrEmpty()) changes.add("operator:wikipedia", answer.operator.wikipedia);
```

Comment:
`operator.wikipedia` is never null or empty, so this check can be removed.

`operator.wikidata` is never null, but may be empty. I'd change the type of that field to `String?` (nullable), and change the `""` there to `null`, so this check can be simplified to `answer.operator.wikidata != null`.

Reply:
> `operator.wikipedia` is never null or empty, so this check can be removed.

Yeh, I just left it for "update" reasons, maybe in future someone might add an operator which might not have a Wikipedia page (very rare but possible :laughing:). But I can remove it, what do you think?

> `operator.wikidata` is never null, but may be empty. I'd change the type of that field to `String?` (nullable), and change the `""` there to null, so this check can be simplified to `answer.operator.wikidata != null`.

Thanks! Finally my doubt has been cleared! I looked it up (maybe with the wrong query) but couldn't find why the IDE gave me that error! That's why I looked up for the `isNullOrEmpty()` function. Again, thank you! Fixing this right now!

Reply:
(I'm very new to Java/Kotlin I don't even know if it's only a Kotlin thing or also a Java one tbh :laughing:)

Reply:
Anyway I just fount the Wikidata tag for "Polizia Locale" so I removed the null check entirely

Reply:
I'm also still learning Kotlin, but I have some Java experience already. In Java, everything can always be `null`, so the explicit "nullable" types are a Kotlin thing. But being able to completely eliminate the null check is of course even better!

Reply:
Perfect, thank you for clarifying! 🤗